src/Components/AuthDialog/AuthDialogContext.tsx

```diff
@@ -40,6 +40,7 @@ export type AuthDialogOptions = {
   title?: string | ((mode: AuthDialogMode) => string)
   onClose?: () => void
   onSuccess?: () => void
+  disableTapToClose?: boolean
```

Comment:
not sure if this is a fine approach

Reply:
Why would we want to do this?

Reply:
IMO, it's annoying and you'll be able to close via escape or the close button anyway. If we want a required auth step then redirect to the full page login/sign up.

Reply:
True, didn't think about keyboard events - I will tell Barney about it. Thanks @dzucconi 🙏

Reply:
Done now - added a redirect to the landing page if the user is not logged in